packages/components/atoms/f-error-message/src/components/ErrorMessage.vue

```diff
@@ -6,6 +6,7 @@
         <danger-icon :class="$style['c-errorMessage-icon']" />
         <span
             data-test-id="content"
+            aria-live="assertive"
```

Comment:
Have you tried `role="alert"` instead? I am just thinking that it is more semantic to use role for the element first and only if there is no appropriate role than to use aria-live

Reply:
Agree, fixed.